case-lib/pipeline.sh

```diff
@@ -2,6 +2,17 @@
 
 func_pipeline_export()
 {
+    # For legacy HDA platform, there is no topology, we have to exporting pipeline
+    # information from proc file system.
+    is_sof_used || {
+        dlogi "No SOF sound card found, exporting pipeline parameters from proc file system"
+        tmp_pipeline_params=/tmp/pipeline_params.sh
```

Comment:
Agreed with @marc-hb to use a mktemp. You can also add prefix if you'd like. Try to play with help.
But I think mktmp will not create same file, so you can leave the delete to the /tmp/ folder clean up
```suggestion
        tmp_pipeline_params=$(mktemp)
```